application/controllers/user.php

```diff
@@ -89,8 +89,10 @@ public function signup() {
             } else {
                 $err_code = '200';
                 unset($data['passconf']);
+                $token = $this->user->set_token($data['mail']);
```

Comment:
It's great to set token in this order, but I prefer we replace `set_token()` to something like `get_token()` and it does nothing to the database. Currently, `set_token()` try to update the database and that not what we want before inserting the user.